tests/data/transformer-module/06-complex-module/authorization-model.json

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
can you change this test case or add another one in which there are relations specific to the relations that are not present in the type